src/org-unit-select/OrgUnitSelectAll.component.js

```diff
@@ -67,12 +69,15 @@ class OrgUnitSelectAll extends React.Component {
     }
 
     getDescendantOrgUnits() {
-        return this.context.d2.models.organisationUnits.list({
-            root: this.props.currentRoot.id,
-            paging: false,
-            includeDescendants: true,
-            fields: "id,path",
-        });
+        return this.context.api.models.organisationUnits
+            .get({
+                root: this.props.currentRoot.id,
```

Comment:
I thought this (using a non-standard _model.get_ field) works in JS because of the lack of typing. In fact, it also works in TS because the machinery necessary to make the get pagination/unpaginated `get` allows passing an extension of the object. That works nicely here, but we would to revisit it property. Issue created: https://github.com/EyeSeeTea/d2-api/issues/19